### ISSUE TITLE
IllegalStateException: Call while result is not pending

### ISSUE THREAD

**Issue by CsabaMiomni** [infobip/mobile-messaging-sdk-android#3]
Can you help me with this please? I'm really confused where can I find the bug.

```
Fatal Exception: java.lang.RuntimeException: Unable to start receiver org.infobip.mobile.messaging.gcm.MobileMessagingGcmReceiver: java.lang.IllegalStateException: Call while result is not pending
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:3194)
       at android.app.ActivityThread.-wrap17(Unknown Source)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1672)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6494)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)

Caused by java.lang.IllegalStateException: Call while result is not pending
       at android.content.BroadcastReceiver.checkSynchronousHint(BroadcastReceiver.java:638)
       at android.content.BroadcastReceiver.abortBroadcast(BroadcastReceiver.java:555)
       at org.infobip.mobile.messaging.gcm.MobileMessagingGcmReceiver.onReceive(SourceFile:41)
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:3187)
       at android.app.ActivityThread.-wrap17(Unknown Source)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1672)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6494)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by CsabaMiomni** [comment]
I'm using the v1.9.1 lib, but maybe the app that's crashing using the v1.7.2 lib.

**Comment by CsabaMiomni** [comment]
Seems it's crashing only on Android 8.

**Comment by stanislavin** [comment]
@CsabaMiomni this should've been fixed in **1.7.8**: https://github.com/infobip/mobile-messaging-sdk-android/releases/tag/1.7.8, please check

**Comment by CsabaMiomni** [comment]
Thank you for the answer, so it is sure that the cirrent app has the older lib. I'll ask the chief to release an update.

Thanks again.
